src/routes/comment.routes.js

```diff
@@ -0,0 +1,118 @@
+
+
+const router = express.Router();
+
+/**
+ * @swagger
+ * tags:
+ *   name: comment
+ *   description: Trips activities
+ */
+
+/**
+ * @swagger
+ *
+ * /trip-request/{tripRequestID}/comments:
+ *    post:
+ *      summary: user and manager can comment to the trip request
+ *      tags: [comment]
+ *      parameters:
+ *       - name: token
+ *         in: header
+ *         description: Check token authentication
+ *         required: true
+ *         type: string
+ *       - name: tripRequestID
+ *         in: path
+ *         description: trip request ID
+ *         required: true
+ *         type: string
+ *      requestBody:
+ *        required: true
+ *        content:
+ *          application/json:
+ *            schema:
+ *              $ref: '#/components/schemas/tripRequestcomment'
+ *      responses:
+ *        "201":
+ *          description: trip request comment schema
+ *
+ * components:
+ *    schemas:
+ *      tripRequestcomment:
+ *        type: object
+ *        required:
+ *          - comment
+ *        properties:
+ *          comment:
+ *            type: string
+ */
+router.post('/:tripRequestID/comments', verifyToken.headerToken, Validate.CommentValidation(), isValid, commentMiddleware.validateTipIdAndUserId, comment.createTripRequestComment);
+
+/**
+ * @swagger
+ * tags:
+ *   name: comment
+ *   description: Trips activities
+ */
+
+/**
+ * @swagger
+ *
+ * /trip-request/{tripRequestID}/comments?page:
+ *    get:
+ *      summary: user and manager can view the trip request comment of
+ *      tags: [comment]
+ *      parameters:
+ *       - name: token
+ *         in: header
+ *         description: Check token authentication
+ *         required: true
+ *         type: string
+ *       - name: tripRequestID
+ *         in: path
+ *         description: trip request ID
+ *         required: true
+ *         type: string
+ *       - name: page
+ *         in: query
+ *         schema:
+ *           type: integer
+ *         description: limit number
+ *         required: true
+ *      responses:
+ *        "200":
+ *          description: trip request comment schema
+ */
+router.get('/:tripRequestID/comments', verifyToken.headerToken, commentMiddleware.validateTipIdAndUserId, comment.getAllComments);
+
+/**
+ * @swagger
+ *
+ * /trip-request/{commentId}/comments:
+ *    delete:
+ *      summary: user and manager can delete the trip request comment
+ *      tags: [comment]
+ *      parameters:
+ *       - name: token
+ *         in: header
+ *         description: Check token authentication
+ *         required: true
+ *         type: string
+ *       - name: commentId
+ *         in: path
+ *         description: comment ID
+ *         required: true
+ *         type: string
+ *      responses:
+ *        "200":
+ *          description: trip request comment schema
+ */
+router.delete('/:commentId/comments', verifyToken.headerToken, commentMiddleware.deleteCommentValidation, comment.deleteComment);
```

Comment:
When deleting a comment of a trip-request, 
the route should be 
"trip-requests/:tripRequestID/comments/:commentID"